docs/configuration.asciidoc

```diff
@@ -674,6 +674,48 @@ When this setting is `true`, the agent will also add the header `elasticapm-trac
 [[config-stacktrace]]
 === Stacktrace configuration options
 [float]
+
+[[config-application-namespaces]]
+==== `ApplicationNamespaces (added[1.5])`
+
+Used to determine whether a stack trace frame is an in-app frame or a library frame. When defined, all namespaces that do not start with one of the values of this collection are ignored when determining error culprit.
+
+Multiple namespaces can be configured as a comma separated list. E.g.: `"MyAppA, MyAppB"`.
```

Comment:
"e.g." is acceptable, but when it's the only part of the sentence, "for example" should be preferred.

```suggestion
Multiple namespaces can be configured as a comma separated list. For example: `"MyAppA, MyAppB"`.
```